allensdk/brain_observatory/behavior/session_apis/data_io/behavior_nwb_api.py

```diff
@@ -54,6 +54,11 @@ def save(self, session_object):
         nwb.add_stimulus_timestamps(nwbfile,
                                     session_object.stimulus_timestamps)
 
+        # Add running acquisition ('dx', 'v_sig', 'v_in') data to NWB
+        nwb.add_running_acquisition_to_nwbfile(
+            nwbfile,
+            session_object.api.get_running_acquisition_df())
```

Comment:
@njmei  can you please explain why `.api` instead of just calling the session object property?

Reply:
Running acquisition data (`dx`, `v_in`, `v_sig`) should be saved to the NWB file, but should not be displayed/accessible at the Session level.

Reply:
Would you mind adding a note to that effect since it differs from how other data is written to NWB.

Reply:
This is done